uint32[] point_serials
float32 length
float32 para_A
float32 para_B
float32 para_C
